src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_CommandBar.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using Uno.UI.RuntimeTests.Helpers;
+using Windows.Foundation.Metadata;
+using Windows.UI;
+using Windows.UI.Xaml.Media.Imaging;
+using static Private.Infrastructure.TestServices;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls;
+
+[TestClass]
+[RunsOnUIThread]
+internal class Given_CommandBar
+{
+	[TestMethod]
+	public async Task TestNativeCommandBarIcon()
+	{
+		if (!ApiInformation.IsTypePresent("Windows.UI.Xaml.Media.Imaging.RenderTargetBitmap"))
+		{
+			Assert.Inconclusive(); // System.NotImplementedException: RenderTargetBitmap is not supported on this platform.;
+		}
+
+		var SUT = new CommandBarTests();
+		WindowHelper.WindowContent = SUT;
+		await WindowHelper.WaitForIdle();
+
+		var renderer = new RenderTargetBitmap();
+		await renderer.RenderAsync(SUT);
+		var result = await RawBitmap.From(renderer, SUT);
```

Comment:
We really need to move this functionality to an external package so we can use it in Toolkit and other places too for the runtime tests, maybe the runtimetestengine?

Reply:
Yes, it would make sense to move it there, though Uno does not use that external control yet but should. I recall @MartinZikmund had made PR for that.